index.js

```diff
@@ -0,0 +1,41 @@
+const initializeChameleonBuilder = (token, { forceOverride } = {}) => {
+  if (forceOverride) {
+    chmln = undefined;
+  } else if (typeof chmln !== 'undefined') {
+    console.log(`Failed to instantiate the global "chmln" variable to initiate the chameleon builder.\nPlease make sure you don\'t have anything assigned to that or use the force param to override any previous declaration.\n\ne.g.   chmln.init('token', { forceOverride: true });`);
```

Comment:
I want to improve those logs actually, maybe `throw Error`? Any dev wants to make a simple PR after this one and take the opportunity to learn how to deploy it to npm? I'm ok with changing this in this CR without problem though

Reply:
maybe a simple console.error?